code/src/WijmoProvider/Columns/DropdownColumn.ts

```diff
@@ -116,10 +120,13 @@ namespace WijmoProvider.Column {
                 this.config.dropdownOptions &&
                 this.config.dropdownOptions.length > 0
             ) {
-                this.changeDisplayValues();
-                if (!this._handlerAdded) {
-                    this._parentHandler();
-                    this._handlerAdded = true;
+                // eslint-disable-next-line no-extra-boolean-cast
+                if (!!this.config.parentBinding) {
```

Comment:
What does this mean? (!!)

Reply:
Use !! to cast to boolean and check if it is true